core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -94,6 +95,7 @@ class KafkaApis(val requestChannel: RequestChannel,
                 val metadataSupport: MetadataSupport,
                 val replicaManager: ReplicaManager,
                 val groupCoordinator: GroupCoordinator,
+                val newGroupCoordinator: org.apache.kafka.coordinator.group.GroupCoordinator,
```

Comment:
This is a temporary change. When all the API are migrated to new new interface, I will change `groupCoordinator` to the new interface and remove this one.